test/integration/sample30-default-order-by.ts

```diff
@@ -0,0 +1,97 @@
+import "reflect-metadata";
```

Comment:
after I added test for this in `functional`, the test failed. https://github.com/typeorm/typeorm/pull/621